website/package.json

```diff
@@ -33,21 +32,17 @@
 	"devDependencies": {
 		"@types/classnames": "^2.3.1",
 		"@types/glob": "^8.1.0",
-		"@types/html-webpack-plugin": "^3.2.2",
 		"@types/react": "^17.0.3",
 		"@types/react-dom": "^17.0.3",
 		"@types/webpack": "^4.41.10",
 		"clean-webpack-plugin": "^3.0.0",
 		"copy-webpack-plugin": "^11.0.0",
 		"css-loader": "^3.5.1",
-		"file-loader": "^6.0.0",
 		"glob": "^9.2.1",
-		"html-webpack-plugin": "^5.5.0",
-		"raw-loader": "^4.0.2",
```

Comment:
the raw-loader is deprecated in Webpack 5.
Instead of the raw-loader must be used `asset/source` and file must be required with the `?raw` query.